cobalt/black_box_tests/testdata/pointer_event_test_utils.js

```diff
@@ -0,0 +1,69 @@
+// Copyright 2023 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file provides test utility functions for pointer event related
+// tests.
+
+// Fail if the test is not finished within 30 seconds.
+const kTimeout = 30 * 1000;
+const failTimer = setTimeout(fail, kTimeout);
```

Comment:
Is failTimer actually used? I think we can just call setTimeout without assigning it to a variable

Reply:
Well, the code all just coming from the existing pointer_test.html, but yeah, will fix these issues.